lib/engine/part/icon.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require_relative 'base'
+
+module Engine
+  module Part
+    class Icon < Base
+      attr_accessor :preprinted
+      attr_reader :image, :name, :sticky
+
+      def initialize(image, name = nil, sticky = true, preprinted = true)
+        @image = "/icons/#{image}.svg"
```

Comment:
you need to update nginx

Reply:
would be nice if you could also DRY up my nginx config, i couldn't figure out how to OR multiple image paths

Reply:
hmm, guessing you tried separating the paths with `|` like the line `location ~ \.(xml|png|ico|webmanifest) {`?

Can't get my local prod stack working, so I can't test out nginx changes right now. I've got these values set in my environment:

```
export POSTGRES_DB=18xx_production
export POSTGRES_PASSWORD=secret
export POSTGRES_USER=18xx
```
Getting this error:

```
queue_1  | Sequel failed to connect: PG::ConnectionBad: FATAL:  password authentication failed for user "18xx"
```